airgun/views/host.py

```diff
@@ -226,7 +226,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixinPF4):
         column_widgets={
             0: Checkbox(locator=".//input[@class='host_select_boxes']"),
             'Name': Text(
-                "//a[contains(@href, '/new/hosts/') and not(contains(@href, 'Insights'))]"
+                ".//a[contains(@href, '/new/hosts/') and not(contains(@href, 'Insights'))]"
```

Comment:
@pnovotny if I remember correctly, this PR can be closed in favor of https://github.com/SatelliteQE/airgun/pull/1468 ?

Reply:
@pnovotny Can you confirm?

Reply:
Sorry for late reply @Gauravtalreja1 
PR https://github.com/SatelliteQE/airgun/pull/1468 is a different issue, but connected to this one.
This PR fixes host name lookup issue when having multiple hosts in the table, but it also exposed another issue there, fixed by the https://github.com/SatelliteQE/airgun/pull/1468. 
Having https://github.com/SatelliteQE/airgun/pull/1468 now merged, re-introducing this fix for the Name locator should solve the problem with multiple hosts, while not breaking the navigation to the host details page.